Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -57,15 +57,32 @@ bool LogFileReader::readAll(const QString& fileName)
  * that the conversion from string to double is sucessful.*/
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
+    bool conversionFailed = true;
```

Comment:
So if conversionFailed was true, wouldn't that imply that the conversion failed? It look like when this is _false_, it means the conversion failed. 

I would rename it to something like, `conversionOk`